src/components/responsive-flyout/styles/ResponsiveFlyout.scss

```diff
@@ -0,0 +1,37 @@
+@import '../../../styles/variables';
+@import '../../../styles/constants/media-queries';
+
+.flyout-overlay {
+    .overlay {
+        .bdl-OverlayHeader {
+            display: none;
+        }
+    }
+}
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .flyout-overlay.responsive-flyout {
+            // cancels out the tether styling without having to set enabled=false
+            transform: none !important;
+
+            .overlay {
+                position: fixed;
+                top: 0;
+                right: 0;
+                bottom: 0;
+                left: 0;
+                margin: 0;
+                padding: 0;
+                background-color: $white;
+                border: none;
+                border-radius: 0;
+                box-shadow: none;
+
+                .bdl-OverlayHeader {
```

Comment:
It looks like this can be moved out of the `.overlay` wrapper to reduce its specificity.